### DISCUSSION TITLE
Getting started with Geist

### DISCUSSION THREAD

**Discussion by verbose9** [zeit-ui/react#468] [Q&A] [answered]
Geist UI has one of the most worst documented website. 
They have not even gave a starting point nor an example, NOTHING.

Can anyone help to kickstart my very first project?

**Comment by unix** [comment]
The first page of the document contains a link to a project that you may have missed, try looking at this one: https://github.com/geist-org/react-getting-started

**Reply by verbose9** [reply]
Nice joke, but not helpful....

No reference to styled component values....

**Comment by agoldis** [comment]
Is it a joke? Can't believe this topic is real. 👎🏻 🙅🏻‍♂️

**Comment by verbose9** [comment]
oh really? 😂

**Comment by verbose9** [comment]
oh really? 😂

**Comment by wizzymore** [comment]
Dude are you joking?

https://react.geist-ui.dev/en-us/guide/introduction

**Comment by ofekashery** [comment, marked as answer]
Sorry, but the language in this discussion does not match the [Geist Code of Conduct](https://github.com/geist-org/react/blob/master/.github/CODE_OF_CONDUCT.md) and unacceptable here, we're going to lock the conversation.

Anyway, the [Geist documentation for React.js](https://react.geist-ui.dev/) explains well how to [get started and installing Geist](https://react.geist-ui.dev/en-us/guide/installation), how to customize [themes](https://react.geist-ui.dev/en-us/guide/themes) and [colors](https://react.geist-ui.dev/en-us/guide/colors), and [how to use each component](https://react.geist-ui.dev/en-us/components).

We also provide [a sample getting started project](https://github.com/geist-org/react-getting-started), and a few more [usage examples](https://github.com/geist-org/react/tree/master/examples).

If you still have a question about Geist, or comments on our documentation, we would love to hear in a new discussion that meets the [Geist Code of Conduct](https://github.com/geist-org/react/blob/master/.github/CODE_OF_CONDUCT.md).
